setup.cfg

```diff
@@ -32,7 +32,7 @@ install_requires =
     psycopg2
     django-better-admin-arrayfield
     django-solo
-    mozilla-django-oidc >=1.0.0, <2.0.0
+    mozilla-django-oidc >=1.0.0, <=2.0.0
```

Comment:
this blocks patch releases like 2.0.1 - I'd specify `~= 2.0.0` which covers patch releases and/or maybe even relax it to `< 3.0` to cover all new minor releases which _should_ be backwards compatible according to semver (but then certain settings/options they add may not be supported yet)

Reply:
Changed this in https://github.com/maykinmedia/mozilla-django-oidc-db/pull/38/commits/5a5d5f0ad82d752abb548c62cbbcf4b83d834ec6